exchange/exchange.go

```diff
@@ -497,6 +507,15 @@ func (e *exchange) getAllBids(
 	return adapterBids, adapterExtra, bidsFound
 }
 
+func insertStoredImpData(impsToStoredRequest map[string][]byte, bids *pbsOrtbSeatBid) {
+	for _, bid := range bids.bids {
+		if val, ok := impsToStoredRequest[bid.bid.ImpID]; ok {
+			result, _ := jsonparser.Set(bid.bid.Ext, val, StoredRequestAttributes)
```

Comment:
Don't forget to remove the `jsonparser.Set`. :) I also think we may need to transform the data, please see this example:

```
"ext": {
    "prebid": {
        "type": "video",
        ...
    },
    "storedrequestattributes": {
        "maxduration": 60,
        "mimes": [
            "video/mp4"
        ],
        "minduration": 15,
        "protocols": [
            1,
            2
        ],
        "skipafter": 0,
        "skipmin": 0,
        "startdelay": 0,
        "playbackmethod": [1]
    }
}
```

Is that processed or is that exactly from the stored imp?

Reply:
Right, modified to marshal/unmarshal logic. Please take another look. 

With current implementation I just write the entire stored impression back to bid.ext, no processing. 
What kind of processing do you mean?